README.md

```diff
@@ -115,6 +116,38 @@ config :logger, :console,
 
 Make sure that the `Plexy.RequestId` plug is included in your Elixir app per the Installation instructions, and you will have the `request_id` in the log metadata.
 
+By default Plexy looks for an ENV variable named `APP_NAME` and will then fall back to `plexy` if it is not provided. This value is used for metrics logging.
+
+```
+Plexy.Logger.measure(:fast, 123)
+# logs => measure#plexy.fast=123
```

Comment:
FWIW: This format is interaly documented [here]( https://github.com/heroku/engineering-docs/blob/master/guides/logs-as-data.md#distributions-measure) and publicly documented [here](https://devcenter.heroku.com/articles/librato#custom-distributions).

Reply:
I called that out as you may want to link to one or both of them. Up to you.

Reply:
Because this is open source I am reluctant to add either, but if I added one it would be the librato doc, the other doc is not available publicly.